kod/object/item/passitem/defmod/armor/robe/robedisc.kod

```diff
@@ -279,6 +288,13 @@ messages:
          Send(poOwner,@RemoveOverlay,#what=self);
       }
 
+      piItem_flags = Send(SYS, @EncodeTwoColorXLAT,
```

Comment:
This always sets all item flags, even those unrelated to colors.  Just double-checking that's what you want.

Reply:
No, I don't want that and didn't realize there were other flags used by that property besides color.  In light of that, it looks like SetPaletteTranslation should always be used instead of setting the property directly going forward.  All fixed.

Reply:
piItem_flags is only for color as far as I know. Any other properties are typically itematt passive objects